other/concat-exports.js

```diff
@@ -0,0 +1,88 @@
+const appendToFile = require('fs').appendFile
```

Comment:
I'm getting no warnings. I think async execution is not super useful here, so I can switch to sync.